samples/drivers/watchdog/boards/stm32h7_wwdg.overlay

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021 Thomas Stranger
```

Comment:
Ok to keep this, but you need to add yours as well

Reply:
adding `Copyright (c) 2022 STMicroelectronics`  on the next line